--- ../textEditor.pro	2023-11-20 20:53:08.389935000 +0800
+++ ../textEditor.pro	2023-11-20 21:22:37.685888316 +0800
@@ -12,3 +12,31 @@
 HEADERS += finddialog.h mainwindow.h
 SOURCES += finddialog.cpp main.cpp mainwindow.cpp
 RESOURCES += resource.qrc
+SVG
+
+
+# desktop指定
+DESKTOP_FILE = ./textEditor.desktop
+system("echo '[Desktop Entry]' > $$DESKTOP_FILE")
+system("echo 'Version=0.2.0' >> $$DESKTOP_FILE")
+system("echo 'Type=Application' >> $$DESKTOP_FILE")
+system("echo 'Name=textEditor' >> $$DESKTOP_FILE")
+system("echo 'Comment=textEditor.' >> $$DESKTOP_FILE")
+system("echo 'Exec=$$PREFIX/bin/textEditor' >> $$DESKTOP_FILE")
+system("echo 'Icon=app' >> $$DESKTOP_FILE")
+system("echo 'Terminal=false' >> $$DESKTOP_FILE")
+system("echo 'Categories=Utility;' >> $$DESKTOP_FILE")
+
+desktop.files += $$DESKTOP_FILE
+desktop.path = $$PREFIX/share/applications
+INSTALLS += desktop
+
+# bin文件指定
+target.path = $$PREFIX/bin
+INSTALLS += target
+
+icon.files = $$PWD/images/app.png
+icon.path = $$PREFIX/share/icons/textEditor/
+INSTALLS += icon
+
+
