--- ../qsnapshot.pro	2023-12-01 09:49:55.562615000 +0800
+++ ../qsnapshot.pro	2023-12-01 09:51:58.094442988 +0800
@@ -30,3 +30,21 @@
 RESOURCES += \
     icons.qrc
 
+DESKTOP_FILE = ./cutesnapshot.desktop
+system("echo '[Desktop Entry]' > $$DESKTOP_FILE")
+system("echo 'Version=1.0.0' >> $$DESKTOP_FILE")
+system("echo 'Type=Application' >> $$DESKTOP_FILE")
+system("echo 'Name=cutesnapshot' >> $$DESKTOP_FILE")
+system("echo 'Comment=cutesnapshot.' >> $$DESKTOP_FILE")
+system("echo 'Exec=cutesnapshot' >> $$DESKTOP_FILE")
+system("echo 'Terminal=false' >> $$DESKTOP_FILE")
+system("echo 'Categories=Utility;' >> $$DESKTOP_FILE")
+
+desktop.files += $$DESKTOP_FILE
+desktop.path = $$PREFIX/share/applications
+INSTALLS += desktop
+
+target.path = $$PREFIX/bin
+INSTALLS += target
+
+
