From 93bc21642ac1c35dc7f28c4f24f19aa9a8612d55 Mon Sep 17 00:00:00 2001
From: aizhuzhudegua <1648476050@qq.com>
Date: Fri, 10 Nov 2023 16:11:27 +0800
Subject: [install.patch_] install

---
 qtmind.pro | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/qtmind.pro b/qtmind.pro
index 1f6779c..5eee720 100644
--- a/qtmind.pro
+++ b/qtmind.pro
@@ -106,7 +106,7 @@ OTHER_FILES += \
 	src/src.qbs
 
 unix:!macx { # installation on Unix-ish platforms
-	isEmpty(INSTALL_PREFIX):INSTALL_PREFIX = /usr
+	isEmpty(INSTALL_PREFIX):INSTALL_PREFIX = $${PREFIX}
 	isEmpty(BIN_DIR):BIN_DIR = $$INSTALL_PREFIX/bin
 	isEmpty(DATA_DIR):DATA_DIR = $$INSTALL_PREFIX/share
 	isEmpty(ICON_DIR):ICON_DIR = $$DATA_DIR/pixmaps
-- 
2.33.1

