diff --git a/qtnotify.desktop b/qtnotify.desktop
new file mode 100644
index 0000000..465e23b
--- /dev/null
+++ b/qtnotify.desktop
@@ -0,0 +1,6 @@
+[Desktop Entry]
+Version=1.0
+Type=Application
+Name=qtnotify
+Exec=qtnotify
+Terminal=false
\ No newline at end of file
diff --git a/qtnotify.pro b/qtnotify.pro
index 0c7237c..60327b1 100644
--- a/qtnotify.pro
+++ b/qtnotify.pro
@@ -22,5 +22,16 @@ SOURCES += \
 RESOURCES += \
     res.qrc
 
+unix: {
+
+target.path = $${PREFIX}/bin
+INSTALLS += target
+
+unix_desktop.path = $${PREFIX}/share/applications
+unix_desktop.files = qtnotify.desktop
+INSTALLS += unix_desktop
+
+}
+
 
 
