diff -Naur b/DrawingTool.desktop a/DrawingTool.desktop
--- b/DrawingTool.desktop	1970-01-01 08:00:00.000000000 +0800
+++ a/DrawingTool.desktop	2024-01-25 21:35:57.848128441 +0800
@@ -0,0 +1,5 @@
+[Desktop Entry]
+Name=DrawingTool
+Exec=DrawingTool
+Icon=/screenshots/screenshot-001.png
+Type=Application
diff -Naur b/DrawingTool.pro a/DrawingTool.pro
--- b/DrawingTool.pro	2024-01-25 21:21:55.092048000 +0800
+++ a/DrawingTool.pro	2024-01-25 21:26:24.423543141 +0800
@@ -47,3 +47,10 @@
 qnx: target.path = /tmp/$${TARGET}/bin
 else: unix:!android: target.path = /opt/$${TARGET}/bin
 !isEmpty(target.path): INSTALLS += target
+
+BINDIR  = $$PREFIX/bin
+DATADIR = $$PREFIX/share
+target.path = $$BINDIR
+desktop.files = DrawingTool.desktop
+desktop.path = $$DATADIR/applications/
+INSTALLS += target desktop
\ 文件尾没有换行符
