--- ../gamescene.cpp	2023-12-05 13:25:36.580718000 +0800
+++ ../gamescene.cpp	2023-12-05 09:53:22.569081000 +0800
@@ -353,17 +353,17 @@
     }
     else if(scoreText.length() == 2) // 10 - 99
     {
-        unityPartVal = scoreText.last(1).toInt();
-        decimalPartValue = scoreText.first(1).toInt();
+        unityPartVal = scoreText.at(scoreText.length() - 1).digitValue();
+        decimalPartValue = scoreText.at(0).digitValue();
         hendredthPartValue = 0;
     }
     else if(scoreText.length() == 3) // 100 - 999
     {
-        unityPartVal = scoreText.last(1).toInt();
-        hendredthPartValue = scoreText.first(1).toInt();
+        unityPartVal = scoreText.at(scoreText.length() - 1).digitValue();
+        hendredthPartValue = scoreText.at(0).digitValue();
         QString copyVal = scoreText;
         copyVal.chop(1);
-        decimalPartValue = copyVal.last(1).toInt();
+        decimalPartValue = copyVal.at(scoreText.length() - 1).digitValue();
     }
 
     unityPartScoreItem.setPixmap(m_numbersPixmap.copy(unityPartVal*32, 0, 32, 32));
