--- ../Bejeweled.pro	2023-12-05 13:25:21.128432000 +0800
+++ ../Bejeweled.pro	2023-12-05 13:29:24.103971000 +0800
@@ -15,3 +15,18 @@
 
 RESOURCES += \
     resource.qrc
+target.path = $$PREFIX/bin # /opt/$${TARGET}/bin
+INSTALLS += target
+
+DESKTOP_FILE = ./Bejeweled.desktop
+system("echo '[Desktop Entry]' > $$DESKTOP_FILE")
+system("echo 'Type=Application' >> $$DESKTOP_FILE")
+system("echo 'Name=Bejeweled' >> $$DESKTOP_FILE")
+system("echo 'Comment=Bejeweled.' >> $$DESKTOP_FILE")
+system("echo 'Exec=Bejeweled' >> $$DESKTOP_FILE")
+system("echo 'Terminal=false' >> $$DESKTOP_FILE")
+system("echo 'Categories=Utility;' >> $$DESKTOP_FILE")
+
+desktop.files += $$DESKTOP_FILE
+desktop.path = $$PREFIX/share/applications
+INSTALLS += desktop
