diff -Naur b/game_PVZ.pro a/game_PVZ.pro
--- b/game_PVZ.pro	2023-12-07 16:59:01.079577648 +0800
+++ a/game_PVZ.pro	2023-12-07 17:01:27.312051981 +0800
@@ -2,7 +2,7 @@
 
 greaterThan(QT_MAJOR_VERSION, 4): QT += widgets
 
-TARGET = game_PVZ
+TARGET = PVZ
 TEMPLATE = app
 
 SOURCES += \
@@ -27,3 +27,14 @@
     button.h \
     sun.h \
     mymap.h
+BINDIR = $$PREFIX/bin
+
+DATADIR = $$PREFIX/share
+
+target.path = $$BINDIR
+
+desktop.files = PVZ.desktop
+
+desktop.path = $$DATADIR/applications/
+
+INSTALLS += target desktop
diff -Naur b/PVZ.desktop a/PVZ.desktop
--- b/PVZ.desktop	1970-01-01 08:00:00.000000000 +0800
+++ a/PVZ.desktop	2023-12-07 17:06:32.589720824 +0800
@@ -0,0 +1,7 @@
+[Desktop Entry]
+
+Exec=PVZ
+
+Name=PVZ
+
+Type=Application
