From bd68d95243f96a08bfdb4e6a1165b3e402e69649 Mon Sep 17 00:00:00 2001
From: van <751890223@qq.com>
Date: Sun, 14 Apr 2024 15:48:27 +0800
Subject: [PATCH] fix-desktop

---
 easybench.pro | 11 +++++++++++
 1 file changed, 11 insertions(+)

diff --git a/easybench.pro b/easybench.pro
index 5c06d81..fe89d69 100644
--- a/easybench.pro
+++ b/easybench.pro
@@ -72,3 +72,14 @@ RESOURCES += \
     resource/stylesheet.qrc
 
 OTHER_FILES +=
+
+
+#install role
+BINDIR  = $$PREFIX/bin
+DATADIR = $$PREFIX/share
+target.path = $$BINDIR
+desktop.files = easybench.desktop 
+icon.files=easybench.png
+icon.path=$$DATADIR/icons/
+desktop.path = $$DATADIR/applications/
+INSTALLS += target desktop icon
\ No newline at end of file
-- 
2.33.1

