diff -Naur b/MahjongPointer.desktop a/MahjongPointer.desktop
--- b/MahjongPointer.desktop	1970-01-01 08:00:00.000000000 +0800
+++ a/MahjongPointer.desktop	2023-12-03 23:07:58.970281645 +0800
@@ -0,0 +1,4 @@
+[Desktop Entry]
+Type=Application
+Name=MahjongPointer
+Exec=MahjongPointer
\ 文件尾没有换行符
diff -Naur b/MahjongPointer.pro a/MahjongPointer.pro
--- b/MahjongPointer.pro	2023-12-03 23:05:30.390110626 +0800
+++ a/MahjongPointer.pro	2023-12-03 23:08:08.346338588 +0800
@@ -1,6 +1,8 @@
 QT       += core gui
 
 greaterThan(QT_MAJOR_VERSION, 4): QT += widgets
+TARGET = MahjongPointer
+TEMPLATE = app
 
 # Only available in Qt 5.14 / Qt 5.15
 !equals(QT_MAJOR_VERSION,5) | lessThan(QT_MINOR_VERSION, 14){
@@ -64,3 +66,10 @@
 
 RESOURCES += \
     src.qrc
+
+
+desktop.files += MahjongPointer.desktop
+desktop.path = $${PREFIX}/share/applications
+target.path = $$(PREFIX)/bin
+
+INSTALLS += target desktop
