From 5f60f83759d78fb5f8bcc3960026f184fd855461 Mon Sep 17 00:00:00 2001
From: van <751890223@qq.com>
Date: Tue, 7 Nov 2023 00:25:28 +0800
Subject: [PATCH] fix-install

---
 CMakeLists.txt | 13 +++++++++++++
 1 file changed, 13 insertions(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index e4312c5..793046e 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -34,3 +34,16 @@ set(SOURCES
 
 add_executable(qreversi ${SOURCES})
 target_link_libraries(qreversi Qt5::Widgets)
+
+set(DESKTOP_FILE_CONTENT "
+[Desktop Entry]
+Type=Application
+Name=qreversi.desktop
+Exec=qreversi
+Categories=Utility;
+")
+
+file(WRITE ${CMAKE_BINARY_DIR}/qreversi.desktop "${DESKTOP_FILE_CONTENT}")
+
+install(PROGRAMS ${CMAKE_BINARY_DIR}/qreversi.desktop DESTINATION share/applications)
+install(TARGETS qreversi DESTINATION bin)
\ No newline at end of file
-- 
2.33.1

