--- ../third-party/catch2/catch.hpp	2023-12-04 10:27:55.317066000 +0800
+++ ../third-party/catch2/catch.hpp	2023-12-04 11:08:11.870695270 +0800
@@ -10788,7 +10788,7 @@
 
     // 32kb for the alternate stack seems to be sufficient. However, this value
     // is experimentally determined, so that's not guaranteed.
-    static constexpr std::size_t sigStackSize = 32768 >= MINSIGSTKSZ ? 32768 : MINSIGSTKSZ;
+    static const std::size_t sigStackSize=32768;
 
     static SignalDefs signalDefs[] = {
         { SIGINT,  "SIGINT - Terminal interrupt signal" },
@@ -10813,6 +10813,7 @@
     }
 
     FatalConditionHandler::FatalConditionHandler() {
+
         isSet = true;
         stack_t sigStack;
         sigStack.ss_sp = altStackMem;
@@ -10844,6 +10845,8 @@
         }
     }
 
+    // 在类外初始化静态成员变量
+//    std::size_t Catch::FatalConditionHandler::sigStackSize; // 注意：这里不再是 constexpr
     bool FatalConditionHandler::isSet = false;
     struct sigaction FatalConditionHandler::oldSigActions[sizeof(signalDefs)/sizeof(SignalDefs)] = {};
     stack_t FatalConditionHandler::oldSigStack = {};
