diff -Naur b/elosi.desktop a/elosi.desktop
--- b/elosi.desktop	1970-01-01 08:00:00.000000000 +0800
+++ a/elosi.desktop	2023-12-06 16:44:34.575690215 +0800
@@ -0,0 +1,3 @@
+[Desktop Entry]
+Exec=elosi
+Name=elosi
diff -Naur b/elosi.pro a/elosi.pro
--- b/elosi.pro	2023-12-06 16:40:25.918636296 +0800
+++ a/elosi.pro	2023-12-06 16:43:20.140980021 +0800
@@ -23,3 +23,11 @@
 qnx: target.path = /tmp/$${TARGET}/bin
 else: unix:!android: target.path = /opt/$${TARGET}/bin
 !isEmpty(target.path): INSTALLS += target
+
+BINDIR = $$PREFIX/bin
+DATADIR = $$PREFIX/share
+target.path = $$BINDIR
+desktop.files =elosi.desktop
+desktop.path = $$DATADIR/applications/
+INSTALLS += target desktop
+
