--- ../segmentation-depthmap-3d-opencv.pro	2023-12-03 18:01:42.813118000 +0800
+++ ../segmentation-depthmap-3d-opencv.pro	2023-12-03 18:06:36.681862374 +0800
@@ -38,3 +38,21 @@
 
 # icons
 RESOURCES += resources.qrc
+
+DESKTOP_FILE = ./segmentation-depthmap-3d-opencv.desktop
+system("echo '[Desktop Entry]' > $$DESKTOP_FILE")
+system("echo 'Version=1.5.0' >> $$DESKTOP_FILE")
+system("echo 'Type=Application' >> $$DESKTOP_FILE")
+system("echo 'Name=segmentation-depthmap-3d-opencv' >> $$DESKTOP_FILE")
+system("echo 'Comment=segmentation-depthmap-3d-opencv.' >> $$DESKTOP_FILE")
+system("echo 'Exec=segmentation-depthmap-3d-opencv' >> $$DESKTOP_FILE")
+system("echo 'Terminal=false' >> $$DESKTOP_FILE")
+system("echo 'Categories=Utility;' >> $$DESKTOP_FILE")
+
+desktop.files += $$DESKTOP_FILE
+desktop.path = $$PREFIX/share/applications
+INSTALLS += desktop
+
+target.path = $$PREFIX/bin
+INSTALLS += target
+
