diff --git a/src/3rdparty/3rdparty.pro b/src/3rdparty/3rdparty.pro
index 55e5271680..3205be912f 100644
--- a/src/3rdparty/3rdparty.pro
+++ b/src/3rdparty/3rdparty.pro
@@ -19,8 +19,10 @@ else {
 }
 
 !r_mobile {
-    exists(qt-labs-qtscriptgenerator-$${QT_VERSION}) {
-        SUBDIRS += qt-labs-qtscriptgenerator-$${QT_VERSION}
+#    exists(qt-labs-qtscriptgenerator-$${QT_VERSION}) {
+#        SUBDIRS += qt-labs-qtscriptgenerator-$${QT_VERSION}
+    exists(qt-labs-qtscriptgenerator-5.15.8) {
+        SUBDIRS += qt-labs-qtscriptgenerator-5.15.8
     }
     else {
         lessThan(QT_MAJOR_VERSION, 6) {
