From a3e3b040939ff11d1198fbeab694cc85a5d4a976 Mon Sep 17 00:00:00 2001
From: van <751890223@qq.com>
Date: Sun, 26 Nov 2023 21:15:15 +0800
Subject: [PATCH] fix-lib

---
 CMakeLists.txt | 2 ++
 1 file changed, 2 insertion(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 8e92afa..26f0f27 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,5 +1,6 @@
 cmake_minimum_required(VERSION 2.8.12...3.5)
 
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fPIC")
+set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -fPIC")
 file(STRINGS ${CMAKE_CURRENT_SOURCE_DIR}/config/version  _VERSION_MAJOR_STRING REGEX "GLEW_MAJOR[ ]*=[ ]*[0-9]+.*")
 string(REGEX REPLACE "GLEW_MAJOR[ ]*=[ ]*([0-9]+)" "\\1" MAJOR_VERSION ${_VERSION_MAJOR_STRING})
 file(STRINGS ${CMAKE_CURRENT_SOURCE_DIR}/config/version  _VERSION_MINOR_STRING REGEX "GLEW_MINOR[ ]*=[ ]*[0-9]+.*")
-- 
2.33.1

