diff --git a/TestLayout.pro b/TestLayout.pro
index 64b0778..ec9ad91 100644
--- a/TestLayout.pro
+++ b/TestLayout.pro
@@ -73,3 +73,10 @@ OBJECTS_DIR = $${BUILD_DIRECTORY}
 MOC_DIR = $${BUILD_DIRECTORY}
 RCC_DIR = $${BUILD_DIRECTORY}
 UI_DIR = $${BUILD_DIRECTORY}
+
+desktop.path = $$PREFIX/share/applications
+desktop.files += build/TestLayout.desktop
+INSTALLS += desktop
+
+target.path = $$PREFIX/bin
+INSTALLS += target
