diff --git a/dist/linux/cutechess.desktop b/dist/linux/cutechess.desktop
index 65e476d..22a2f7d 100644
--- a/dist/linux/cutechess.desktop
+++ b/dist/linux/cutechess.desktop
@@ -2,8 +2,8 @@
 Name=Cute Chess
 Comment=Interface for playing chess
 GenericName=Chess Game
-Icon=cutechess
-Exec=cutechess
+Icon=cutechess_512x512
+Exec=cutegomoku %f
 Terminal=false
 Type=Application
 Categories=Game;BoardGame;
diff --git a/projects/gui/gui.pro b/projects/gui/gui.pro
index a0dfb14..33aa1a2 100644
--- a/projects/gui/gui.pro
+++ b/projects/gui/gui.pro
@@ -47,3 +47,14 @@ include(3rdparty/qcustomplot/qcustomplot.pri)
 CONFIG(debug, debug|release) {
     include(3rdparty/modeltest/modeltest.pri)
 }
+
+desktop.path = $$PREFIX/share/applications
+desktop.files += ../../dist/linux/cutechess.desktop
+INSTALLS += desktop
+
+target.path = $$PREFIX/bin
+INSTALLS += target
+
+icons.path = $${PREFIX}/share/icons
+icons.files = res/icons/cutechess_512x512.png
+INSTALLS += icons
