diff --git a/GnuplotViewerQuick.desktop b/GnuplotViewerQuick.desktop
new file mode 100644
index 0000000..ecbbc51
--- /dev/null
+++ b/GnuplotViewerQuick.desktop
@@ -0,0 +1,5 @@
+[Desktop Entry]
+Type=Application
+Name=Gnuplot Viewer Quick
+Exec=GnuplotViewerQuick %f
+Icon=icon64x64
diff --git a/GnuplotViewerQuick.pro b/GnuplotViewerQuick.pro
index 7f63a7d..8fd6293 100644
--- a/GnuplotViewerQuick.pro
+++ b/GnuplotViewerQuick.pro
@@ -60,8 +60,17 @@ QML_DESIGNER_IMPORT_PATH =
 # Default rules for deployment.
 qnx: target.path = /tmp/$${TARGET}/bin
 else: unix:!android: target.path = /opt/$${TARGET}/bin
+target.path = $$PREFIX/bin
 !isEmpty(target.path): INSTALLS += target
 
+desktop.path = $$PREFIX/share/applications
+desktop.files += GnuplotViewerQuick.desktop
+INSTALLS += desktop
+
+icons.path = $${PREFIX}/share/icons
+icons.files = gnuplot/src/wxterminal/bitmaps/xpm/icon64x64.xpm
+INSTALLS += icons
+
 HEADERS += \
     androidtasks.h \
     applicationdata.h \

