--- ../app/app.pro	2023-11-23 09:33:22.459131000 +0800
+++ ../app/app.pro	2023-11-11 16:43:30.407502000 +0800
@@ -51,7 +51,20 @@
 
 # Default rules for deployment.
 qnx: target.path = /tmp/$${TARGET}/bin
-else: unix:!android: target.path = /opt/$${TARGET}/bin
+else: unix:!android: target.path = $$PREFIX/bin
 !isEmpty(target.path): INSTALLS += target
 
 win32:RC_ICONS = res/icon.ico
+
+DESKTOP_FILE = ./hexed.desktop
+system("echo '[Desktop Entry]' > $$DESKTOP_FILE")
+system("echo 'Type=Application' >> $$DESKTOP_FILE")
+system("echo 'Name=hexed' >> $$DESKTOP_FILE")
+system("echo 'Comment=hexed.' >> $$DESKTOP_FILE")
+system("echo 'Exec=hexed' >> $$DESKTOP_FILE")
+system("echo 'Terminal=false' >> $$DESKTOP_FILE")
+system("echo 'Categories=Utility;' >> $$DESKTOP_FILE")
+
+desktop.files += $$DESKTOP_FILE
+desktop.path = $$PREFIX/share/applications
+INSTALLS += desktop
