--- ../QCalcFileHash.pro	2023-12-18 11:20:14.817079000 +0800
+++ ../QCalcFileHash.pro	2023-12-18 11:26:07.650883000 +0800
@@ -64,27 +64,27 @@
 updateqm.CONFIG += no_link target_predeps
 QMAKE_EXTRA_COMPILERS += updateqm
 
-data_bin.path = /usr/bin/
+data_bin.path = $$PREFIX/bin/
 data_bin.files = Bin/qcalcfilehash
 INSTALLS += data_bin
 
-data_app.path = /usr/share/applications/
+data_app.path = $$PREFIX/share/applications/
 data_app.files = pkg/qcalcfilehash.desktop
 INSTALLS += data_app
 
-data_langs.path = /usr/share/qcalcfilehash/langs/
+data_langs.path = $$PREFIX/share/qcalcfilehash/langs/
 data_langs.files = langs/*.qm langs/langs.json
 INSTALLS += data_langs
 
-data_icons16.path = /usr/share/icons/hicolor/16x16/apps/
+data_icons16.path = $$PREFIX/share/icons/hicolor/16x16/apps/
 data_icons16.files = pkg/icons/16x16/qcalcfilehash.png
 INSTALLS += data_icons16
 
-data_icons32.path = /usr/share/icons/hicolor/32x32/apps/
+data_icons32.path = $$PREFIX/share/icons/hicolor/32x32/apps/
 data_icons32.files = pkg/icons/32x32/qcalcfilehash.png
 INSTALLS += data_icons32
 
-data_icons48.path = /usr/share/icons/hicolor/48x48/apps/
+data_icons48.path = $$PREFIX/share/icons/hicolor/48x48/apps/
 data_icons48.files = pkg/icons/48x48/qcalcfilehash.png
 INSTALLS += data_icons48
 
