--- ../Luxmetre.pro	2023-12-19 19:16:02.605305000 +0800
+++ ../Luxmetre.pro	2023-12-19 19:14:48.006302000 +0800
@@ -25,5 +25,17 @@
 
 # Default rules for deployment.
 qnx: target.path = /tmp/$${TARGET}/bin
-else: unix:!android: target.path = /opt/$${TARGET}/bin
+else: unix:!android: target.path = $$PREFIX/bin
 !isEmpty(target.path): INSTALLS += target
+DESKTOP_FILE = ./Luxmetre.desktop
+system("echo '[Desktop Entry]' > $$DESKTOP_FILE")
+system("echo 'Type=Application' >> $$DESKTOP_FILE")
+system("echo 'Name=Luxmetre' >> $$DESKTOP_FILE")
+system("echo 'Comment=Luxmetre.' >> $$DESKTOP_FILE")
+system("echo 'Exec=Luxmetre' >> $$DESKTOP_FILE")
+system("echo 'Terminal=false' >> $$DESKTOP_FILE")
+system("echo 'Categories=Utility;' >> $$DESKTOP_FILE")
+
+desktop.files += $$DESKTOP_FILE
+desktop.path = $$PREFIX/share/applications
+INSTALLS += desktop
