--- ../meson.build	2023-10-30 11:03:38.374912000 +0800
+++ ../meson.build	2023-10-30 11:54:38.104867000 +0800
@@ -14,7 +14,7 @@
 )
 
 add_project_arguments(
-  '-Wno-implicit-fallthrough',
+  '-Wno--fallthrough',
   '-Wno-dangling-else',
   '-Wno-newline-eof',
   '-Wno-unknown-pragmas',
@@ -34,6 +34,20 @@
 subdir('lib')
 subdir('res')
 
+
+desktop_contents = '''
+[Desktop Entry]
+Name=orbvis
+Exec=orbvis
+Icon=orbvis
+Type=Application
+Categories=Utility;
+'''
+
+configure_file(output: 'orbvis.desktop',  configuration: {'DESKTOP_CONTENTS': desktop_contents})
+
+install_data('/source/build/orbvis.desktop', install_dir: '/opt/apps/io.github.orbvis/files/share/applications' )
+
 executable(
   'orbvis',
   orbvis_sources,
@@ -42,4 +56,5 @@
   include_directories : [orbvis_incdir, lib_incdir],
   dependencies : deps,
   install : true,
+  install_dir: '/opt/apps/io.github.orbvis/files/bin'
 )
