diff --git a/QCommandEdit.pro b/QCommandEdit.pro
index ff63a81..412fdf3 100644
--- a/QCommandEdit.pro
+++ b/QCommandEdit.pro
@@ -26,3 +26,10 @@ DEFINES += \
     QT_DISABLE_DEPRECATED_BEFORE=0x060000 \
     QT_RESTRICTED_CAST_FROM_ASCII \
     QT_NO_KEYWORDS
+
+desktop.path = $$PREFIX/share/applications
+desktop.files += QCommandEdit.desktop
+INSTALLS += desktop
+
+target.path = $$PREFIX/bin
+INSTALLS += target
