diff -Naur b/LilyDict.desktop a/LilyDict.desktop
--- b/LilyDict.desktop	1970-01-01 08:00:00.000000000 +0800
+++ a/LilyDict.desktop	2023-12-08 13:44:43.735873781 +0800
@@ -0,0 +1,3 @@
+[Desktop Entry]
+Exec=LilyDict
+Name=LilyDict
diff -Naur b/LilyDict.pro a/LilyDict.pro
--- b/LilyDict.pro	2023-12-08 13:31:50.595003000 +0800
+++ a/LilyDict.pro	2023-12-08 13:43:11.028424274 +0800
@@ -43,7 +43,12 @@
 #DEFINES += QT_NO_WARNING_OUTPUT\
 #           QT_NO_DEBUG_OUTPUT
 
-
+BINDIR = $$PREFIX/bin
+DATADIR = $$PREFIX/share
+target.path = $$BINDIR
+desktop.files = LilyDict.desktop
+desktop.path = $$DATADIR/applications/
+INSTALLS += target desktop
 
 
 
