diff --git a/CMakeLists.txt b/CMakeLists.txt
index edff589..c4cfe2b 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -33,3 +33,8 @@ target_link_libraries(${PROJECT_NAME}
     Qt5::MultimediaWidgets
     Qt5::Widgets
 )
+
+
+
+install(TARGETS QtBasedVideoEditor DESTINATION bin)
+install(PROGRAMS veido.desktop DESTINATION share/applications)
diff --git a/veido.desktop b/veido.desktop
new file mode 100644
index 0000000..53b5f27
--- /dev/null
+++ b/veido.desktop
@@ -0,0 +1,6 @@
+[Desktop Entry]
+Version=1.0
+Type=Application
+Name=Qt-based-video-editor
+Exec=QtBasedVideoEditor
+Terminal=false
\ No newline at end of file
