--- ../QtSnake/QtSnake.pro	2023-11-10 17:45:37.306590000 +0800
+++ ../QtSnake/QtSnake.pro	2023-11-10 17:52:48.646831326 +0800
@@ -16,3 +16,20 @@
         widget.cpp
 
 HEADERS  += widget.h
+
+DESKTOP_FILE = ./QtSnake.desktop
+system("echo '[Desktop Entry]' > $$DESKTOP_FILE")
+system("echo 'Version=1.0.0' >> $$DESKTOP_FILE")
+system("echo 'Type=Application' >> $$DESKTOP_FILE")
+system("echo 'Name=QtSnake' >> $$DESKTOP_FILE")
+system("echo 'Comment=QtSnake, a tool.' >> $$DESKTOP_FILE")
+system("echo 'Exec=$$PREFIX/bin/QtSnake' >> $$DESKTOP_FILE")
+system("echo 'Terminal=false' >> $$DESKTOP_FILE")
+system("echo 'Categories=Utility;' >> $$DESKTOP_FILE")
+
+desktop.files += $$DESKTOP_FILE
+desktop.path = $$PREFIX/share/applications
+INSTALLS += desktop
+
+target.path = $$PREFIX/bin
+INSTALLS += target
