--- ../qmidiplayer-desktop/CMakeLists.txt	2023-11-02 19:31:16.941820000 +0800
+++ ../qmidiplayer-desktop/CMakeLists.txt	2023-12-02 10:21:16.885672486 +0800
@@ -98,9 +98,22 @@
 install(DIRECTORY ${PROJECT_SOURCE_DIR}/doc DESTINATION ${CMAKE_INSTALL_PREFIX}/share/qmidiplayer)
 install(DIRECTORY ${PROJECT_SOURCE_DIR}/img DESTINATION ${CMAKE_INSTALL_PREFIX}/share/qmidiplayer FILES_MATCHING PATTERN "*.png")
 install(FILES qmidiplayer.appdata.xml DESTINATION ${CMAKE_INSTALL_PREFIX}/share/appdata)
-install(FILES qmidiplayer.desktop DESTINATION ${CMAKE_INSTALL_PREFIX}/share/applications)
+# install(FILES qmidiplayer.desktop DESTINATION ${CMAKE_INSTALL_PREFIX}/share/applications)
 install(FILES ${PROJECT_SOURCE_DIR}/img/qmidiplyr.png DESTINATION ${CMAKE_INSTALL_PREFIX}/share/icons/hicolor/64x64/apps)
 install(FILES ${PROJECT_SOURCE_DIR}/img/qmidiplyr.svg DESTINATION ${CMAKE_INSTALL_PREFIX}/share/icons/hicolor/scalable/apps)
 install(FILES menu/qmidiplayer DESTINATION ${CMAKE_INSTALL_PREFIX}/share/menu)
 install(FILES qmidiplayer.mime DESTINATION ${CMAKE_INSTALL_PREFIX}/share/mime/packages)
 install(FILES ${qmpdesktop_QM_FILES} DESTINATION ${CMAKE_INSTALL_PREFIX}/share/qmidiplayer/translations)
+
+# 写入desktop
+set(DESKTOP_FILE_CONTENT "
+[Desktop Entry]
+Type=Application
+Name=qmidiplayer
+Exec=qmidiplayer
+Icon=qmidiplyr
+Categories=Utility;
+")
+file(WRITE ${CMAKE_INSTALL_PREFIX}/share/applications/qmidiplayer.desktop "${DESKTOP_FILE_CONTENT}")
+
+
