--- ../arkanoid.pro	2023-12-17 14:30:34.506831000 +0800
+++ ../arkanoid.pro	2023-12-17 14:28:36.671829000 +0800
@@ -60,8 +60,19 @@
 
 # Default rules for deployment.
 qnx: target.path = /tmp/$${TARGET}/bin
-else: unix:!android: target.path = /opt/$${TARGET}/bin
+else: unix:!android: target.path = $$PREFIX/bin
 !isEmpty(target.path): INSTALLS += target
+DESKTOP_FILE = ./arkanoid.desktop
+system("echo '[Desktop Entry]' > $$DESKTOP_FILE")
+system("echo 'Type=Application' >> $$DESKTOP_FILE")
+system("echo 'Name=arkanoid' >> $$DESKTOP_FILE")
+system("echo 'Comment=arkanoid.' >> $$DESKTOP_FILE")
+system("echo 'Exec=arkanoid' >> $$DESKTOP_FILE")
+system("echo 'Terminal=false' >> $$DESKTOP_FILE")
+system("echo 'Categories=Utility;' >> $$DESKTOP_FILE")
 
+desktop.files += $$DESKTOP_FILE
+desktop.path = $$PREFIX/share/applications
+INSTALLS += desktop
 RESOURCES += \
     res.qrc
