--- ../app/app.pro	2023-11-09 21:16:00.368134000 +0800
+++ ../app/app.pro	2023-11-09 21:26:22.377948963 +0800
@@ -37,3 +37,20 @@
 CONFIG       += c++14
 RESOURCES    += images.qrc
 TRANSLATIONS += trans_pl.ts
+
+DESKTOP_FILE = ./app.desktop
+system("echo '[Desktop Entry]' > $$DESKTOP_FILE")
+system("echo 'Version=1.0.0' >> $$DESKTOP_FILE")
+system("echo 'Type=Application' >> $$DESKTOP_FILE")
+system("echo 'Name=app' >> $$DESKTOP_FILE")
+system("echo 'Comment=app, a tool.' >> $$DESKTOP_FILE")
+system("echo 'Exec=$$PREFIX/bin/app' >> $$DESKTOP_FILE")
+system("echo 'Terminal=false' >> $$DESKTOP_FILE")
+system("echo 'Categories=Utility;' >> $$DESKTOP_FILE")
+
+desktop.files += $$DESKTOP_FILE
+desktop.path = $$PREFIX/share/applications
+INSTALLS += desktop
+
+target.path = $$PREFIX/bin
+INSTALLS += target
