diff -Naur b/build-aux/meson/postinstall.py a/build-aux/meson/postinstall.py
--- b/build-aux/meson/postinstall.py	2023-11-20 18:45:28.507450101 +0800
+++ a/build-aux/meson/postinstall.py	2023-11-20 18:47:19.143782746 +0800
@@ -12,8 +12,8 @@
     print('Updating icon cache...')
     call(['gtk-update-icon-cache', '-qtf', path.join(datadir, 'icons', 'hicolor')])
 
-    print('Updating desktop database...')
-    call(['update-desktop-database', '-q', path.join(datadir, 'applications')])
+   # print('Updating desktop database...')
+  #  call(['update-desktop-database', '-q', path.join(datadir, 'applications')])
 
     print('Compiling GSettings schemas...')
     call(['glib-compile-schemas', path.join(datadir, 'glib-2.0', 'schemas')])
diff -Naur b/diffuse.desktop a/diffuse.desktop
--- b/diffuse.desktop	1970-01-01 08:00:00.000000000 +0800
+++ a/diffuse.desktop	2023-11-20 18:46:50.102963205 +0800
@@ -0,0 +1,4 @@
+[Desktop Entry]
+Name=diffuse
+Exec=diffuse
+Type=Application
diff -Naur b/meson.build a/meson.build
--- b/meson.build	2023-11-20 18:45:28.515447341 +0800
+++ a/meson.build	2023-11-20 19:14:28.330513099 +0800
@@ -13,3 +13,5 @@
 if build_machine.system() == 'linux'
   meson.add_install_script('build-aux/meson/postinstall.py')
 endif
+
+install_data('diffuse.desktop', install_dir : '$(PREFIX)/share/application')
