--- ../demo.pro	2023-12-09 15:22:27.564568000 +0800
+++ ../demo.pro	2023-12-09 15:21:26.404279000 +0800
@@ -21,5 +21,18 @@
 
 # Default rules for deployment.
 qnx: target.path = /tmp/$${TARGET}/bin
-else: unix:!android: target.path = /opt/$${TARGET}/bin
+else: unix:!android: target.path = $$PREFIX/bin
 !isEmpty(target.path): INSTALLS += target
+
+DESKTOP_FILE = ./demo.desktop
+system("echo '[Desktop Entry]' > $$DESKTOP_FILE")
+system("echo 'Type=Application' >> $$DESKTOP_FILE")
+system("echo 'Name=demo' >> $$DESKTOP_FILE")
+system("echo 'Comment=demo.' >> $$DESKTOP_FILE")
+system("echo 'Exec=demo' >> $$DESKTOP_FILE")
+system("echo 'Terminal=false' >> $$DESKTOP_FILE")
+system("echo 'Categories=Utility;' >> $$DESKTOP_FILE")
+
+desktop.files += $$DESKTOP_FILE
+desktop.path = $$PREFIX/share/applications
+INSTALLS += desktop
