From a1b246f66b0c642680f4cbfb6831871901899643 Mon Sep 17 00:00:00 2001
From: van <751890223@qq.com>
Date: Wed, 17 Apr 2024 22:30:00 +0800
Subject: [PATCH] fix-install

---
 meson.build | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/meson.build b/meson.build
index af4a6bcd97..f314519ef8 100644
--- a/meson.build
+++ b/meson.build
@@ -1650,7 +1650,7 @@ if win32
 endif
 
 
-if get_option('libmpv')
+if true
     client_h_define = cc.get_define('MPV_CLIENT_API_VERSION', prefix: '#include "libmpv/client.h"',
                                     include_directories: include_directories('.'))
     major = client_h_define.split('|')[0].split('<<')[0].strip('() ')
@@ -1668,7 +1668,7 @@ if get_option('libmpv')
     install_headers(headers, subdir: 'mpv')
 endif
 
-if get_option('cplayer')
+if false
     datadir = get_option('datadir')
     confdir = get_option('sysconfdir')
 
-- 
2.33.1

