diff -Naur b/SerialDebuger_QCustomPlot.pro a/SerialDebuger_QCustomPlot.pro
--- b/SerialDebuger_QCustomPlot.pro	2024-01-25 23:17:03.348300000 +0800
+++ a/SerialDebuger_QCustomPlot.pro	2024-01-25 23:22:13.933949124 +0800
@@ -4,6 +4,9 @@
 
 CONFIG += c++11
 
+TEMPTLATE = app
+
+TARGET = SerialPlot
 # The following define makes your compiler emit warnings if you use
 # any Qt feature that has been marked deprecated (the exact warnings
 # depend on your compiler). Please consult the documentation of the
@@ -41,3 +44,10 @@
 
 RESOURCES += \
     res.qrc
+
+BINDIR  = $$PREFIX/bin
+DATADIR = $$PREFIX/share
+target.path = $$BINDIR
+desktop.files = SerialPlot.desktop
+desktop.path = $$DATADIR/applications/
+INSTALLS += target desktop
\ 文件尾没有换行符
diff -Naur b/SerialPlot.desktop a/SerialPlot.desktop
--- b/SerialPlot.desktop	1970-01-01 08:00:00.000000000 +0800
+++ a/SerialPlot.desktop	2024-01-25 23:26:15.736495294 +0800
@@ -0,0 +1,5 @@
+[Desktop Entry]
+Name=SerialPlo
+Exec=SerialPlot
+Icon=debug_32px_1103189_easyicon.net.ico
+Type=Application
