--- ../CMakeLists.txt	2023-12-08 16:32:08.872629000 +0800
+++ ../CMakeLists.txt	2023-12-08 16:43:50.884656205 +0800
@@ -33,8 +33,17 @@
 target_link_libraries(${PROJECT_NAME} Qt5::Widgets)
 
 
-install(TARGETS ${PROJECT_NAME} DESTINATION /opt/ezp_editor)
-install(FILES "${CMAKE_CURRENT_SOURCE_DIR}/other/ezp_editor.desktop" DESTINATION /opt/ezp_editor)
-install(FILES "${CMAKE_CURRENT_SOURCE_DIR}/other/ezp_editor.desktop" DESTINATION /usr/share/applications)
-install(FILES "${CMAKE_CURRENT_SOURCE_DIR}/img/chipEdit64.png" DESTINATION /opt/ezp_editor)
-install(FILES "${CMAKE_CURRENT_SOURCE_DIR}/languages/chipEditor_ru_RU.qm" DESTINATION /opt/ezp_editor/languages)
+install(TARGETS ${PROJECT_NAME} DESTINATION ${CMAKE_INSTALL_PREFIX}/bin)
+# install(FILES "${CMAKE_CURRENT_SOURCE_DIR}/other/ezp_editor.desktop" DESTINATION ${CMAKE_INSTALL_PREFIX}/ezp_editor)
+set(DESKTOP_FILE_CONTENT "
+[Desktop Entry]
+Type=Application
+Name=ezp_editor
+Exec=ezp_editor
+Icon='chipEdit64.png'
+Categories=Utility;
+")
+file(WRITE ${CMAKE_INSTALL_PREFIX}/share/applications/ezp_editor.desktop "${DESKTOP_FILE_CONTENT}")
+# install(FILES "${CMAKE_CURRENT_SOURCE_DIR}/other/ezp_editor.desktop" DESTINATION ${CMAKE_INSTALL_PREFIX}/share/applications)
+install(FILES "${CMAKE_CURRENT_SOURCE_DIR}/img/chipEdit64.png" DESTINATION ${CMAKE_INSTALL_PREFIX}/share/icons/ezp_editor)
+install(FILES "${CMAKE_CURRENT_SOURCE_DIR}/languages/chipEditor_ru_RU.qm" DESTINATION ${CMAKE_INSTALL_PREFIX}/ezp_editor/languages)
