From fbdcd582fca891c1f6b53a197f950404079317d0 Mon Sep 17 00:00:00 2001
From: wjyrich <1071633242@qq.com>
Date: Fri, 24 Nov 2023 10:45:28 +0800
Subject: [PATCH] install

---
 libs/ui/CMakeLists.txt | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/libs/ui/CMakeLists.txt b/libs/ui/CMakeLists.txt
index a88c7b7..2e0d1e2 100644
--- a/libs/ui/CMakeLists.txt
+++ b/libs/ui/CMakeLists.txt
@@ -483,6 +483,8 @@ target_link_libraries(kritaui KF5::CoreAddons KF5::Completion KF5::I18n KF5::Ite
                       kritacolor kritaimage kritalibbrush kritawidgets kritawidgetutils ${PNG_LIBRARIES} ${EXIV2_LIBRARIES}
 )
 
+target_link_libraries(kritaui exiv2 ${EXIV2_LIBRARIES})
+
 if (HAVE_KIO)
     target_link_libraries(kritaui KF5::KIOCore)
 endif() 
-- 
2.33.1

