--- ./CMakeLists.txt	2023-10-28 15:22:17.980999000 +0800
+++ ./CMakeLists.txt	2023-11-01 13:51:27.375260567 +0800
@@ -60,6 +60,16 @@
 		ARCHIVE DESTINATION lib
 		)
 
+set(DESKTOP_FILE_CONTENT "
+[Desktop Entry]
+Type=Application
+Name=${CMAKE_PROJECT_NAME}
+Exec=${CMAKE_PROJECT_NAME}
+Categories=Utility;
+")
+file(WRITE ${CMAKE_PROJECT_NAME}/share/applications/PrestoPalette.desktop "${DESKTOP_FILE_CONTENT}")
+install(TARGETS ${CMAKE_PROJECT_NAME} BUNDLE DESTINATION /bin)
+
 IF(MINGW)
 	function(force_find_library file_name)
 		SET(FOUND_LIB "FOUND_LIB-NOTFOUND")
