diff --git a/source b/source
new file mode 160000
index 0000000..bb56235
--- /dev/null
+++ b/source
@@ -0,0 +1 @@
+Subproject commit bb5623538c0df226406336298fbe467e39857b6f
diff --git a/src/src.pro b/src/src.pro
index 0f02798..e8e70d9 100644
--- a/src/src.pro
+++ b/src/src.pro
@@ -4,7 +4,7 @@ qtHaveModule(printsupport): QT += printsupport
 
 greaterThan(QT_MAJOR_VERSION, 4): QT += widgets
 
-# For use with Qt 6
+# For use with Qt 6  
 # CONFIG += c++11
 
 TARGET = aphototoollibre
@@ -14,6 +14,7 @@ CONFIG += warn_on
 # TRANSLATIONS = i18n/aphototoollibre_en_US.ts
 # RESOURCES = i18n/aphototoollibre_en_US.qm
 
+
 DESTDIR = ../bin
 MOC_DIR = ../build/moc
 RCC_DIR = ../build/rcc
@@ -116,22 +117,22 @@ RESOURCES += \
 
 unix: {
 
-target.path = /usr/bin/
+target.path = $${PREFIX}/bin
 INSTALLS += target
 
-unix_man.path = /usr/share/man/man1
+unix_man.path = $${PREFIX}/share/man/man1
 unix_man.files = ../debian/aphototoollibre.1.gz
 INSTALLS += unix_man
 
-unix_desktop.path = /usr/share/applications
+unix_desktop.path = $${PREFIX}/share/applications
 unix_desktop.files = ../desktop/aphototoollibre.desktop
 INSTALLS += unix_desktop
 
-unix_icons.path = /usr/share/icons/hicolor/scalable/apps
+unix_icons.path = $${PREFIX}/share/icons/hicolor/scalable/apps
 unix_icons.files = ../desktop/aphototoollibre.svg
 INSTALLS += unix_icons
 
-unix_metainfo.path = /usr/share/metainfo
+unix_metainfo.path = $${PREFIX}/share/metainfo
 unix_metainfo.files = ../desktop/aphototoollibre.metainfo.xml
 INSTALLS += unix_metainfo
 
