--- ../CMakeLists.txt	2023-11-11 19:27:33.571893000 +0800
+++ ../CMakeLists.txt	2023-11-11 19:28:21.916542000 +0800
@@ -22,7 +22,18 @@
     message(WARNING "CMAKE_PREFIX_PATH is not defined, you may need to set it "
             "(-DCMAKE_PREFIX_PATH=\"path/to/Qt/lib/cmake\" or -DCMAKE_PREFIX_PATH=/usr/include/{host}/qt{version}/ on Ubuntu)")
 endif ()
+set(DESKTOP_FILE_CONTENT "
+[Desktop Entry]
+Type=Application
+Name=QSystemInfo
+Exec=QSystemInfo
+Categories=Utility;
+")
 
+file(WRITE ${CMAKE_BINARY_DIR}/QSystemInfo.desktop "${DESKTOP_FILE_CONTENT}")
+install(PROGRAMS ${CMAKE_BINARY_DIR}/QSystemInfo.desktop DESTINATION ${CMAKE_INSTALL_PREFIX}/share/applications)
+
+install(TARGETS QSystemInfo DESTINATION ${CMAKE_INSTALL_PREFIX}/bin)
 if ( CMAKE_COMPILER_IS_GNUCC )
     set(CMAKE_CXX_FLAGS  "${CMAKE_CXX_FLAGS} -Wall -Wextra -pedantic -Wunused")
 endif()
