--- ../src/QtGuitar.pro	2023-12-02 19:25:51.381535000 +0800
+++ ../src/QtGuitar.pro	2023-12-02 19:35:49.019902660 +0800
@@ -49,3 +49,21 @@
 
 RESOURCES += \
     resource.qrc
+
+DESKTOP_FILE = ./QtGuitar.desktop
+system("echo '[Desktop Entry]' > $$DESKTOP_FILE")
+system("echo 'Version=1.0.0' >> $$DESKTOP_FILE")
+system("echo 'Type=Application' >> $$DESKTOP_FILE")
+system("echo 'Name=QtGuitar' >> $$DESKTOP_FILE")
+system("echo 'Comment=QtGuitar.' >> $$DESKTOP_FILE")
+system("echo 'Exec=QtGuitar' >> $$DESKTOP_FILE")
+system("echo 'Terminal=false' >> $$DESKTOP_FILE")
+system("echo 'Categories=Utility;' >> $$DESKTOP_FILE")
+
+desktop.files += $$DESKTOP_FILE
+desktop.path = $$PREFIX/share/applications
+INSTALLS += desktop
+
+target.path = $$PREFIX/bin
+INSTALLS += target
+
