diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index f3cf60e..0f58409 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -131,10 +131,11 @@ endif(USE_OPENMPT)
 add_library(${PROJECT_NAME})
 target_link_libraries(${PROJECT_NAME} PUBLIC anmp-internal)
 target_link_libraries(${PROJECT_NAME} PUBLIC ${LD_FLAGS})
-install(TARGETS ${PROJECT_NAME} DESTINATION ${CMAKE_INSTALL_LIBDIR})
+install(TARGETS ${PROJECT_NAME} DESTINATION ${CMAKE_INSTALL_PREFIX}/bin)
 
 
 install(FILES Common/PlaylistFactory.h Common/types.h Common/Event.h Common/Nullable.h DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}/${PROJECT_NAME})
 install(FILES InputLibraryWrapper/SongInfo.h DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}/${PROJECT_NAME})
 install(FILES PlayerLogic/IPlaylist.h PlayerLogic/Playlist.h PlayerLogic/Player.h PlayerLogic/Config.h DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}/${PROJECT_NAME})
+install(FILES ../contrib/anmp.desktop DESTINATION ${CMAKE_INSTALL_PREFIX}/share/applications)

