diff --git a/HotShots.pro b/HotShots.pro
index a0b11e6..b62d092 100644
--- a/HotShots.pro
+++ b/HotShots.pro
@@ -10,11 +10,11 @@ TARGET = hotshots
 TEMPLATE = app
 
 # directories setup
-isEmpty( INSTALL_PREFIX )  : INSTALL_PREFIX  = /usr/local
+isEmpty( INSTALL_PREFIX )  : INSTALL_PREFIX  = $$PREFIX
 
 isEmpty( INSTALL_BINDIR )  : INSTALL_BINDIR  = $$INSTALL_PREFIX/bin
 isEmpty( INSTALL_LIBDIR )  {
-    INSTALL_LIBDIR  = $$INSTALL_PREFIX/lib
+    INSTALL_LIBDIR  = lib/${TRIPLET}
     #installing to lib64 on 64-bit platforms:
     contains (QMAKE_HOST.arch, x86_64) : INSTALL_LIBDIR = $$INSTALL_PREFIX/lib64
 }
diff --git a/hotshots.desktop b/hotshots.desktop
new file mode 100644
index 0000000..43fabfc
--- /dev/null
+++ b/hotshots.desktop
@@ -0,0 +1,10 @@
+[Desktop Entry]
+Version=2.2.0
+Name=HotShots
+Comment=Screenshot manager
+Type=Application
+GenericName=Screenshot manager
+Exec=hotshots %f
+Categories=Utility;Application;
+Icon=hotshots
+MimeType=application/x-hot;
