From dc261efb3cd0696e611e9a710c7ed68dabc6d7f7 Mon Sep 17 00:00:00 2001
From: wjyrich <1071633242@qq.com>
Date: Thu, 7 Dec 2023 09:33:52 +0800
Subject: [PATCH] install
    
---
 CMakeLists.txt | 7 ++-----
 1 file changed, 2 insertions(+), 5 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index cb05792..44e96fd 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -147,15 +147,12 @@ include_directories(${CMAKE_SOURCE_DIR}/include)
 set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++0x -Wall")
 #set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -stdlib=libc++ -std=c++11 -Wall")
 
-if(ENABLE_QT5)
+
 #find_package(Qt5Core REQUIRED)
 	find_package(Qt5 COMPONENTS Core Gui Widgets Network REQUIRED)
 	set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS}    ${Qt5Widgets_EXECUTABLE_COMPILE_FLAGS}")
 #	set(CMAKE_CXX_FLAGS "-fPIC")
-else()
-	# Configure the use of QT4
-	find_package(Qt4 COMPONENTS QtCore QtNetwork QtGui REQUIRED QUIET)
-endif()
+
 
 #add libusb and pthreads
 find_package(libusb-1.0 REQUIRED)
-- 
2.33.1

