diff -Naur b/GameView.pro a/GameView.pro
--- b/GameView.pro	2023-12-03 21:43:17.846530929 +0800
+++ a/GameView.pro	2023-12-03 21:44:51.349109780 +0800
@@ -28,3 +28,9 @@
 
 RESOURCES += \
     resources.qrc
+
+desktop.files += GoMoKuGame.desktop
+desktop.path = $${PREFIX}/share/applications
+target.path = $${PREFIX}/bin
+
+INSTALLS += target desktop
\ 文件尾没有换行符
diff -Naur b/GoMoKuGame.desktop a/GoMoKuGame.desktop
--- b/GoMoKuGame.desktop	1970-01-01 08:00:00.000000000 +0800
+++ a/GoMoKuGame.desktop	2023-12-03 21:46:29.683473117 +0800
@@ -0,0 +1,4 @@
+[Desktop Entry]
+Name=GoMokuGame
+Exec=GameView
+Type=Application
\ 文件尾没有换行符
