From 13554ac481978463c1be10248eee94e5fb96b2ed Mon Sep 17 00:00:00 2001
From: wjyrich <1071633242@qq.com>
Date: Tue, 9 Jan 2024 15:09:06 +0800
Subject: [PATCH] install

---
 src/ProcDepMonitor.pro | 12 +++++++++++-
 1 file changed, 11 insertions(+), 1 deletion(-)

diff --git a/src/ProcDepMonitor.pro b/src/ProcDepMonitor.pro
index f0acbc8..b0dce31 100644
--- a/src/ProcDepMonitor.pro
+++ b/src/ProcDepMonitor.pro
@@ -143,5 +143,15 @@ CONFIG(debug, debug|release) {
 
 # Default rules for deployment.
 qnx: target.path = /tmp/$${TARGET}/bin
-else: unix:!android: target.path = /opt/$${TARGET}/bin
+else: unix:!android: #target.path = /opt/$${TARGET}/bin
 !isEmpty(target.path): INSTALLS += target
+
+
+BINDIR = $$PREFIX/bin
+DATADIR = $$PREFIX/share
+target.path = $$BINDIR
+desktop.files =../deploy/unix/linuxdeployqt/app.desktop
+desktop.path = $$DATADIR/applications/
+icons.path = $$PREFIX/share/icons
+icons.files = ../deploy/unix/linuxdeployqt/icons/hicolor/64x64/apps/procdepmonitor.png
+INSTALLS += target desktop icons
\ No newline at end of file
-- 
2.33.1

