diff --git a/CMakeLists.txt b/CMakeLists.txt
index 7e892215..8abe63dc 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -33,8 +33,8 @@ if (WIN32 OR APPLE)
     endif ()
 else ()
     # i.e. Linux variants
-    set (CMAKE_INSTALL_PREFIX "/usr")
-    set(TBE_BIN_DIR     games)
+    set (CMAKE_INSTALL_PREFIX ${PREFIX})
+    set(TBE_BIN_DIR     bin)
     set(TBE_LEVELS_DIR  share/games/tbe/levels)
     set(TBE_IMAGES_DIR  share/games/tbe/images)
     set(TBE_I18N_DIR    share/games/tbe/i18n)

